pages/index.js

```diff
@@ -1,8 +1,10 @@
 import HomeHeader from "../components/pages/home/HomeHeader";
+import Navbar from "../components/pages/home/Navbar";
```

Comment:
Setelah resolve conflict, saran saya ganti pakai absolute import saja.
```suggestion
import Navbar from "@/components/pages/home/Navbar";
```

Absolute import ada di [jsconfig.json](https://github.com/bellshade/bellshade-website-v2/blob/main/jsconfig.json).

Reply:
@Mgkusumaputra gaskeun

Reply:
eh iya luva bentar ku fix